### ISSUE TITLE
iPC_RailVB Values are 0 when they should be NULL

### ISSUE THREAD

**Issue by tyler1218hatch** [Riverscapes/sqlBRAT#56]
In a watershed without Railroads in the valley bottom like North Fork John Day, all streams are given an iPC_RailVB value of 0. This is then symbolized and interpreted as every stream being right next to a railroad in the valley bottom. This could be fixed by assigning those streams NULL or negative values to distinguish them.

![image](https://user-images.githubusercontent.com/27084463/86185066-192a9480-bafb-11ea-9429-6573befc2371.png)
![image](https://user-images.githubusercontent.com/27084463/86185099-28a9dd80-bafb-11ea-87ec-d3029125339a.png)

**Comment by philipbaileynar** [comment]
Unfortunately, [ShapeFile fields cannot be set to NULL](https://support.esri.com/en/technical-article/000012852), so let's use -1 and update the symbology accordingly.

**Comment by MattReimer** [comment]
Agreed. This is pretty ugly and we have to be careful our -1 values never get used for math but given all that I think it's a good convention to use.
